dags/pytorch_xla/pytorchxla_torchvision.py

```diff
@@ -78,7 +100,8 @@
       US_EAST1_C,
   ).run()
 
-  mnist_v2_8 >> (resnet_v2_8, resnet_v4_8, resnet_v5lp_4)
+  mnist_v2_8 >> (resnet_v2_8, *resnet_v4_8_tests, resnet_v4_32, resnet_v5lp_4)
+  resnet_v2_8 >> resnet_v3_8_tests
 
   resnet_v100_2x2 = task.GpuGkeTask(
```

Comment:
We probably want to add post_process sooner to multihost GPU tests, as our dashboard summary is missing the track of those [tests](http://go/ml-auto-solutions-dashboard).

Reply:
I can add this once I'm done porting tests for the 2.3 release.